### DISCUSSION TITLE
How to put Drawer under Navbar?

### DISCUSSION THREAD

**Discussion by ibnu-ja** [saadeghi/daisyui#712] [Q&A]
Hello!
I'm looking for a way to put drawer under the navigation bar.
There is an example, but I use `v-show` on Menu in Vue to achieve this. It adds `display: none;` to HTML elements. It's something hacky and I'd like to know how to do it without any JS.
Also I'd like normal behavior when in mobile size.

Thank you!

https://user-images.githubusercontent.com/10094582/163195267-ba3a8fe9-0ffd-48e2-ad9e-d3a64241c41f.mp4

**Comment by saadeghi** [comment]
Instead of `v-show`, use `v-model` for the checkbox because the value of checkbox controls the visibility of sidebar

**Reply by ibnu-ja** [reply]
Hi, thank you for your response.
I think my question is a bit unclear. I want sidebar appearance like `drawer-mobile` but also toggleable
I've already bind input checkbox to drawer state, and `drawer-mobile` class to the state
transition is a bit choppy, but it works

Maybe there a way to make them more fluid?

https://user-images.githubusercontent.com/10094582/163235464-ead2a80e-5b28-4d0a-b807-21cbe4f42c66.mp4

**Reply by saadeghi** [reply]
Can you provide a code example?

**Reply by ibnu-ja** [reply]
> Can you provide a code example?

https://stackblitz.com/edit/daisyui-vue-vite-hslknn?file=src/layouts/partials/Drawer.vue

here it is

**Reply by saadeghi** [reply]
Thank you. Unfortunately toggling `drawer-mobile` class was not something I considered when I was making the animations. I don't have a quick and simple solution for it for now but I will consider this case for future updates.

**Reply by NathanTe** [reply]
+1 would be pretty sweet
